api/python/provisioner/inputs.py

```diff
@@ -436,9 +436,9 @@ class PillarInputBase(PillarItemsAPI):
         metadata={
             METADATA_ARGPARSER: {
                 'help': 'pillar value',
-                'type': functools.partial(
-                    AttrParserArgs.value_from_str, v_type='json'
-                )
+                #'type': functools.partial(
```

Comment:
Why do we delete this?

Reply:
This code is failing and we don't need this for recent changes.

Reply:
If its not required delete this block (do not comment)

Reply:
Addressed